### PR TITLE
Make paypal checkout blue

### PR DESCRIPTION
**Pull request by rupertbates**
## Why are you doing this?
The PayPal checkout flow is currently using the old styling whereas the standard flow has been updated to a lovely blue colour.

This PR makes them look the same.

## Changes
* Copy all the styling changes across the standard flow to the PayPal flow

## Screenshots
Before:
screen shot 2017-01-10 at 15 27 32
After:
screen shot 2017-01-12 at 16 05 08

### CODE REVIEW

File: frontend/app/views/fragments/form/cardDetailPayPal.scala.html [hunk 1]
```diff
@@ -36,6 +34,9 @@ <h3 class="credit-card-note__header">First payment</h3>
                 <h3 class="credit-card-note__header">Cancellation</h3>
                 <p>You can cancel your membership online through your profile page.</p>
             </div>
+            <div class="credit-card-note">
+                <h3 class="credit-card-note__header"><a class="js-change-payment-method" href="#">Change payment method</a></h3>
+            </div>
```
Comment (Ap0c) [inline frontend/app/views/fragments/form/cardDetailPayPal.scala.html]: This doesn't show up on mobile for some reason, weird... Perhaps there's a CSS rule hiding it somewhere:

payment-method-mobile

File: frontend/assets/stylesheets/components/_payment-b.scss [hunk 2]
```diff
@@ -173,6 +173,12 @@
       @include mq($from: tablet) {
         display: block;
       }
+
+      a {
+        color: guss-colour(guardian-brand);
+        display: block;
+        margin-top: 10px;
+      }
```
Comment (Ap0c) [inline frontend/assets/stylesheets/components/_payment-b.scss]: Do we still need this as a separate file? The `b` refers to the B part of the AB test that's no longer running doesn't it?

**Comment by Ap0c** [comment]
The `_payment-b.scss` file was originally created for the checkout AB in #1379. Now that this is done, and the changes have been applied, is it worth merging this into `_payment.scss` and losing the references to the B flow (`.skin-checkout-b` for example)? @svillafe what do you think?

This is probably not something for this PR, but just a thought.

File: frontend/app/views/fragments/form/cardDetailPayPal.scala.html (hunk 1 above)
Comment (rupertbates) [inline frontend/app/views/fragments/form/cardDetailPayPal.scala.html]: @Ap0c Fixed now if you'd care to have another look

**Comment by Ap0c** [comment]
Yup, it's there on mobile now, all good 😄

**Pull request merged**